Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: build test
+
+build:
+	go build -i -o artifacts/compete github.com/LukeJelly/Cell-Compete-Go
+
+test:
+	go test -race $(go list ./... | grep -v vendor)
```

Comment:
In a large project we might use a fancier build tool, but `make` is pretty standard.